fv3core/tests/main/test_grid.py

```diff
@@ -262,7 +262,7 @@ def test_domain_compute(
     add: Index3D,
     domain_compute: Index3D,
 ):
-    grid = fv3core.utils.grid.GridIndexing(
+    grid = pace.stencils.testing.grid.GridIndexing(
```

Comment:
is GridIndexing in pace.dsl or pace.stencils.testing.grid?

Reply:
It is `pace.dsl`, updated here and elsewhere